sdk/servicebus/service-bus/test/README.md

```diff
@@ -8,46 +8,44 @@
+| topic-filter-subscription                 | Subscription in the Topic `topic-filter`                                               | TOPIC_FILTER_SUBSCRIPTION_NAME                  |
+| topic-filter-default-subscription         | Subscription in the Topic `topic-filter`                                               | TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME          |
+
+    Following are additional environment variables that will need to be defined in order to be able to run the tests in browser
 
     Entitiy Name  | Entity Description | Environment Variable if not using default names
-  ------------- | ------------------|----------
-  partitioned-queue-browser | Queue with partitions enabled and sessions disabled | QUEUE_NAME_BROWSER
-  unpartitioned-queue-browser | Queue with partitions and sessions, both disabled | QUEUE_NAME_NO_PARTITION_BROWSER
-  partitioned-queue-sessions-browser | Queue with partitions and sessions, both enabled | QUEUE_NAME_SESSION_BROWSER
-  unpartitioned-queue-sessions-browser | Queue with partitions disabled and sessions enabled | QUEUE_NAME_NO_PARTITION_SESSION_BROWSER
-  partitioned-topic-browser | Topic with partitions enabled, meant for testing subscriptions with sessions disabled | TOPIC_NAME_BROWSER
-  unpartitioned-topic-browser | Topic with partitions disabled, meant for testing subscriptions with sessions disabled | TOPIC_NAME_NO_PARTITION_BROWSER
-  partitioned-topic-sessions-browser | Topic with partitions enabled, meant for testing subscriptions with sessions enabled | TOPIC_NAME_SESSION_BROWSER
-  unpartitioned-topic-sessions-browser | Topic with partitions disabled, meant for testing subscriptions with sessions enabled | TOPIC_NAME_NO_PARTITION_SESSION_BROWSER
-  partitioned-topic-subscription-browser | Subscription with sessions disabled in the Topic, `partitioned-topic-browser` | SUBSCRIPTION_NAME_BROWSER
-  unpartitioned-topic-subscription-browser | Subscription with sessions disabled in the Topic, `unpartitioned-topic-browser` | SUBSCRIPTION_NAME_NO_PARTITION_BROWSER
-  partitioned-topic-sessions-subscription-browser | Subscription with sessions enabled in the Topic, `partitioned-topic-sessions-browser` | SUBSCRIPTION_NAME_SESSION_BROWSER
-  unpartitioned-topic-sessions-subscription-browser | Subscription with sessions enabled in the Topic, `unpartitioned-topic-sessions-browser` | SUBSCRIPTION_NAME_NO_PARTITION_SESSION_BROWSER
-  topic-filter-browser | Topic for testing topic filters | TOPIC_FILTER_NAME_BROWSER
-  topic-filter-subscription-browser | Subscription in the Topic `topic-filter-browser` | TOPIC_FILTER_SUBSCRIPTION_NAME_BROWSER
-  topic-filter-default-subscription-browser | Subscription in the Topic `topic-filter-browser` | TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME_BROWSER
 
+------------- | ------------------|----------
```

Comment:
This table is no longer rendering properly in markdown

Reply:
I think the line above it needs its indentation removed.

Reply:
Oh, looks like auto-format messed things up - fixed